docs/_docs/04_reference/built-in-functions.md

```diff
@@ -232,6 +234,31 @@ remote_state {
 
 Note that [Terraform will read environment variables](https://www.terraform.io/docs/configuration/environment-variables.html#tf_var_name) that start with the prefix `TF_VAR_`, so one way to share a variable named `foo` between Terraform and Terragrunt is to set its value as the environment variable `TF_VAR_foo` and to read that value in using this `get_env()` built-in function.
 
+## get\_platform
+
+`get_platform()` returns the current Operational System. Example:
+
+``` hcl
+inputs = {
+  platform = get_platform()
+}
+```
+
+This function can also be used in a comparison to evaluate what to do based on the current operating system. Example:
+``` hcl
+output "platform" {
+  value = var.platform == "darwin" ? "" : ""
```

Comment:
```suggestion
  value = var.platform == "darwin" ? "(value for MacOS)" : "(value for other OS's)"
```

Reply:
yep, It's much better!

I was in doubt to decide how to do a good doc to cover this point.